plugins/node/plugin.zsh

```diff
@@ -0,0 +1,18 @@
+# Color definitions
+GEOMETRY_COLOR_NODE_NPM_VERISON=${GEOMETRY_COLOR_NODE_NPM_VERISON:-black}
+
+# Symbol definitions
+GEOMETRY_SYMBOL_NODE_NPM_VERISON=${GEOMETRY_SYMBOL_NODE_NPM_VERSION:-"⬡"}
+GEOMETRY_NODE_NPM_VERISON=$(prompt_geometry_colorize $GEOMETRY_COLOR_NODE_NPM_VERISON GEOMETRY_SYMBOL_NODE_NPM_VERISON) 
+
+geometry_prompt_node_setup() 
+
+geometry_prompt_node_render(){
+	local node_path="$echo "$(which node)" -v"
+	local npm_path="$echo "$(which npm)" --version"
+	GEOMETRY_NODE_VERSION=`(eval ${node_path})`
```

Comment:
From your explanation, the statement "$+commands[node]" returns an alias "node". Is this right? If so, how to add npm to this array?